app/app.go

```diff
@@ -372,6 +380,9 @@ func New(
 	ibcRouter.AddRoute(icahosttypes.SubModuleName, icahost.NewIBCModule(app.ICAHostKeeper)) // Add ICA route
 	app.IBCKeeper.SetRouter(ibcRouter)
 
+	app.HyperlaneKeeper = hyperlanekeeper.NewKeeper(encodingConfig.Codec, encodingConfig.AddressCodec, runtime.NewKVStoreService(keys[hyperlanetypes.ModuleName]), govModuleAddr, app.BankKeeper)
+	app.WarpKeeper = warpkeeper.NewKeeper(encodingConfig.Codec, encodingConfig.AddressCodec, runtime.NewKVStoreService(keys[warptypes.ModuleName]), govModuleAddr, app.BankKeeper, &app.HyperlaneKeeper, nil)
```

Comment:
Do we know what we (if) should enable here instead of the nil? https://github.com/bcp-innovations/hyperlane-cosmos/blob/main/x/warp/keeper/keeper.go#L128-L136

Reply:
Yeah, great question. I wondered that myself when I put this here but I forgot about it when publishing the PR. 

It's likely that it should be set to something. We can merge and flag on the upstream PR and ask the celestia team